lib/device-sockets/ios/app-debug-socket-proxy-factory.ts

```diff
@@ -152,12 +158,11 @@ export class AppDebugSocketProxyFactory extends EventEmitter implements IAppDebu
 			appDebugSocket.on("close", () => {
 				this.$logger.info("Backend socket closed!");
 				webSocket.close();
-				server.close();
```

Comment:
Is the server socket not closed on purpose?

Reply:
The server socket is `appDebugSocket` and this is it's `close` handler. `server` is our proxy server and it's not closed in order to stay available for new client connections. Because of this line, the Chrome Dev Tools was not able to reconnect when you try to refresh the page.